router/core/graphql_handler.go

```diff
@@ -242,11 +243,6 @@ func (h *GraphQLHandler) preparePlan(requestOperationName []byte, shared *pool.S
 	}, nil
 }
 
-func respondWithInternalServerError(w http.ResponseWriter, requestLogger *zap.Logger) {
```

Comment:
don't understand this change.

Reply:
We had multiple ways to set a status code. I removed this specific way and went with `w.WriteHeader(statusCode)`.